index.js

```diff
@@ -18,35 +18,22 @@ const setRandomColor = () => {
         document.body.style.color = 'white'
             btn.classList.remove('btn-outline-dark')
             btn.classList.add('btn-outline-light')
-        btnCalc.classList.remove('btn-outline-dark')
-        btnCalc.classList.add('btn-outline-light')
         clear.classList.remove('btn-outline-dark')
         clear.classList.add('btn-outline-light')
     } else {
         document.body.style.color = 'black'
         btn.classList.remove('btn-outline-light')
         btn.classList.add('btn-outline-dark')
-        btnCalc.classList.remove('btn-outline-light')
-        btnCalc.classList.add('btn-outline-dark')
         clear.classList.remove('btn-outline-light')
         clear.classList.add('btn-outline-dark')
     }
 }
 
 btn.addEventListener('click', () => setRandomColor())
 
-const calculateLetters = text => text.length
-const calculateWords = text => text.trim().split(' ').filter(e => e !== '').length
-// const calculateSentences = text => text.trim().split('.').length - 1
-const calculateSentences = text => {
-    let count = 0
-    for (let char of text) {
-        if (char === '.' || char === '?'|| char === '!') {
-            count++
-        }
-    }
-    return count
-}
+const calculateLetters = text => text.split(/[A-Za-z]/).length - 1
+const calculateWords = text => text.split(/\S+/).length - 1
```

Comment:
Counts the word afther the space. And if you insert (.) or another and make a space will count the word.

Reply:
Errors fixed pls check again